integrations/Vero/browser.js

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+import get from "get-value";
+import { NAME } from "./constants";
+import logger from "../../utils/logUtil";
+import { isDefinedAndNotNull } from "../utils/commonUtils";
+import ScriptLoader from "../ScriptLoader";
+
+class Vero {
+  constructor(config) {
+    this.apiKey = config.apiKey;
+    this.name = NAME;
+  }
+
+  init() {
+    logger.debug("===In init Vero===");
+    window._veroq = window._veroq || [];
+    ScriptLoader(null, "https://d3qxef4rp70elm.cloudfront.net/m.js", true);
+    window._veroq.push(["init", { api_key: this.apiKey }]);
+  }
+
+  isLoaded() {
+    logger.debug("in Vero isLoaded");
+    return !!window._veroq;
```

Comment:
If the script is successfully loaded, I believe this variable would no longer be an array. We should use something of this sort to assert it is loaded.

Reply:
done

Reply:
This is incorrectly done. @ItsSudip / @utsabc, can you guide him?

Reply:
@sauravlal-rudderstack When Vero script is loaded can you check if there is any `loaded` callback provided by script? example `_veroq.loaded` or check what is the type of this variable by checking in console
```
type of _veroq
```

Reply:
type of _veroq is object.


`window._veroq.ready` is a boolean which is available in the _veroq object and is being used in isReady(), I dont't find any other loaded callback in the script.
CC @utsabc

Reply:
@saikumarrs it looks like when `_veroq` is loaded its type becomes object so would a `object` type check be safe as we don't have a call back here it we have no better option

Reply:
Yeah, that seems fine.